testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -12292,6 +12299,18 @@ void testTimestampDiff(boolean coercionEnabled) {
             + "'TIMESTAMP_TRUNC\\(<TIME\\(0\\)>, <INTERVAL HOUR>\\)'\\. "
             + "Supported form\\(s\\): 'TIMESTAMP_TRUNC\\(<TIMESTAMP>, <DATETIME_INTERVAL>\\)'",
         false);
+    f.checkNull("timestamp_trunc(CAST(NULL AS TIMESTAMP), second)");
```

Comment:
Same comment as above. Maybe one or two of these is good but an exhaustive list doesn't add much value other than making the test methods longer than necessary.